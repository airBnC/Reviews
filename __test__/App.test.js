import React from 'react';
import { configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import App from '../client/src/App';

configure({ adapter: new Adapter() });

describe('App', () => {
  it('should render correctly', () => {
    const output = shallow(<App />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});

