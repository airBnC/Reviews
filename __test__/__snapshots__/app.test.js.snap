// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div>
  <h1>
     Top of App 
  </h1>
  <ContainerAverageStars />
  <h1>
     Break! 
  </h1>
  <ContainerReviews />
  <h1>
     End Module 
  </h1>
</div>
`;
